install_config/router/default_haproxy_router.adoc

```diff
@@ -336,6 +336,55 @@ $ openssl ciphers
 Alternatively, use the `ROUTER_CIPHERS` environment variable for an existing
 router.
 
+[[mutual-tls-auth]]
+== Mutual TLS Authentication
+
+Client access to the router and the backend services can be restricted using
+mutual TLS authentication. The router will reject requests from clients not in
+its `authenticated` set.
+Mutual TLS authentication is implemented on client certificates and can be
+controlled based on the certifying authorities (CAs) issuing the certificates,
+the certificate revocation list and/or any certificate subject filters.
+Use the mutual tls config options `--mutual-tls-auth`, `--mutual-tls-auth-ca`,
+`--mutual-tls-auth-crl` and `--mutual-tls-auth-filter` when creating a router:
+
+----
+$ oc adm router --mutual-tls-auth=required  \
+                --mutual-tls-auth-ca=/local/path/to/cacerts.pem   ....
+----
+
+The `--mutual-tls-auth` values are: `required`, `optional`, or `none`, with
```

Comment:
Is that possible to add more details/examples of the difference between `required` and `optional` ?

Reply:
Done - added a blurb on that.